### PR TITLE
Fetch ratings

### PR DESCRIPTION
**Pull request by hfaerber**
### What's this PR do?
- Sets up initial fetch to get user's movie ratings.
- In the commits you will see that I set up the fetch with `ratings` as a separate key in the store but refactored it to include it as a key in the `user` object that is already in store.  User object starts as `{}` rather already including the keys.  This keeps it consistent when we clear it back to `{}` upon log out.

### Where should the reviewer start?
- Pull down and use the Redux chrome extension to see the update to the user object upon logging in.

### How should this be manually tested?
-I wanted to get this up to you but have not yet built any tests.  I would like to edit/build the Redux tests that are affected by these changes.  I will work on those then push up to this same branch.

### Any background context you want to provide?
- Each rating object in the array has an id (for the rating itself) and a created_at property.  We will need the id to delete any ratings.  We will need the created_at property for an extension that has us sort by this info. 
  
### What are the relevant tickets?
This ticket begins work on issue #6 but does not complete the ticket.

### CODE REVIEW

File: src/containers/LoginForm.js [hunk 1]
```diff
@@ -24,9 +24,12 @@ export class LoginForm extends Component {
     event.preventDefault();
     fetchUserLogin(this.state.email, this.state.password)
       .then(data => {
-        const { updateUser, updateLoggedInStatus } = this.props;
-        updateUser({ ...data.user });
-        updateLoggedInStatus(true);
+        const { updateUser, updateLoggedInStatus, addRatings } = this.props;
+        fetchRatings(data.user.id)
```
Comment (peeratmac) [inline src/containers/LoginForm.js]: @hfaerber I got an error for this, but I think it's a quick fix that we need to import fetchRatings in from apiCalls.

Comment (hfaerber) [reply src/containers/LoginForm.js]: Thanks Peerat!  I lost it on the merge conflict but just added it back with that import corrected.  You may need to pull down again to see it.  Thank you!

Comment (peeratmac) [reply src/containers/LoginForm.js]: Got it, I see the fix!  Everything looks good to me.

**Comment by peeratmac** [comment]
@Dunn-Austen this looks good to me, you can merge once you review this!

**Pull request merged**